tests/format/flow-repo/es6modules/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1516,10 +1516,8 @@ import { specifierNumber } from "ES6_Named1"; // Error: Did you mean \`specifier
 ///////////////////////////////////////////////////
 // == Multi \`export *\` should combine exports == //
 ///////////////////////////////////////////////////
-import {
-  numberValue1 as numberValue8,
-  numberValue2 as numberValue9,
-} from "./ES6_ExportAllFromMulti";
+import { numberValue1 as numberValue8, numberValue2 as numberValue9 }
+  from "./ES6_ExportAllFromMulti";
 
```

Comment:
Hmm not sure I prefer this over the original 🤔

Even where there is no `as`, new formatting makes a bit harder to scan through names. AFAIU, the main benefit we should expect from the change is the reduction of lines of code. But what does it improve in DX? Existing approach optimizes for cognitive ease and clean `git diff` – that will be a shame to lose.

Reply:
Good point about `as`. The current formatting seems to work better for such imports.

As for clean diffs, I think the proposed formatting might be an improvement in that regard. Currently, the diff isn't clean when switching between the single-line and multi-line formatting happens. With the proposed formatting, that switching is seemingly going to happen less often, and the diff for switching single-line⇄double-line is quite clean, definitely cleaner than single⇄multi.

Reply:
Just realized I'm used to diff viewers that highlight in-line diffs like this:
![image](https://user-images.githubusercontent.com/94334/188122113-a9b7f06d-469c-4bf6-98b3-244ba95d2c19.png)
And you're probably talking about raw unhighlighted output of `git diff`. That makes me wonder how many people view diffs without in-line highlighting and why they would do that.

Reply:
> But what does it improve in DX?

Less scrolling through something that is edited mostly by tooling (auto-import, linters for sorting imports and removing unused ones), not manually? I wish we had some statistics on how many developers edit imports manually. That'd put this proposal in a good perspective. I guess these people are already in minority, but that's only a guess.

Reply:
I never need write code more than 200 lines. Split the file, if it's too long, that's what ES Module for.

Reply:
> auto-import

I have these lines in my vscode settings

```js
  "javascript.suggest.autoImports": false,
  "vetur.completion.autoImport": false,
  "volar.completion.autoImportComponent": false,
```

Reply:
> I have these lines in my vscode settings

I'm surprised. To me, this is like formatting code manually instead of using Prettier. The only problem I ever had with auto-import has been [fixed in TS 4.8](https://devblogs.microsoft.com/typescript/announcing-typescript-4-8/#exclude-globs-auto-import). Do you think there are many people who share you preferences in this regard?

Reply:
I don't know.

Reply:
[There are few, seach on github](https://github.com/search?q=javascript.suggest.autoImports&type=code)

Reply:
also didn't know that someone these days moves imports manually)